homeassistant/components/twitter/notify.py

```diff
@@ -72,6 +73,7 @@ def __init__(
     def send_message(self, message="", **kwargs):
         """Tweet a message, optionally with media."""
         data = kwargs.get(ATTR_DATA)
+        target = kwargs.get(ATTR_TARGET)
```

Comment:
According to the service specifications, the target can be a list. It seems like that case isn't handled?